imap_processing/cdf/utils.py

```diff
@@ -166,7 +167,17 @@ def write_cdf(
     if "compression" not in extra_cdf_kwargs:
         extra_cdf_kwargs["compression"] = 6  # type: ignore
 
-    xarray_to_cdf(dataset, str(file_path), **extra_cdf_kwargs)
+    prev_cdflib_level = cdflib_logger.level
```

Comment:
can you rename to something like this?
```suggestion
    cdflib_log_level = cdflib_logger.level
```

Reply:
I want to keep the "prev" in there because the point of this is to keep the old version around so we can reset it after the function call later.